README.md

```diff
@@ -18,4 +18,4 @@ og deretter
 
 ## Releasing (kun for medlemmer av Digipost organisasjonen)
 
-Se docs/systemer/open-source-biblioteker.md
+Se docs/systemer/digipost/klientbiblioteker-java.md
```

Comment:
```suggestion

Se [docs/systemer/digipost/klientbiblioteker-java.md](https://github.com/digipost/docs/systemer/digipost/klientbiblioteker-java.md)
```

Reply:
Nope. Vi skal ikke eksponere intern struktur i lukket kildekode. Endringen til Torbjørn er OK.

Reply:
Altså jeg ser den overordnede grunnen, men jeg er helt uenig. Men det får være en diskusjon for en annen gang.